guides/common/modules/con_redis-tuning.adoc

```diff
@@ -0,0 +1,10 @@
+[id="Redis_Tuning_{context}"]
+= Redis Tuning
+
+Redis is an in-memory data store.
+It is used by multiple services in {Project}.
+Dynflow and Pulp tasking systems use it to track their tasks.
```

Comment:
I think an article is needed here, but not 100% sure since I'm not a native speaker.
```suggestion
The Dynflow and Pulp tasking systems use it to track their tasks.
```